dokka-subprojects/plugin-base-frontend/create-component.mjs

```diff
@@ -0,0 +1,63 @@
+import fs from 'fs';
+
+const firstLetterToLoweCase = str => str.charAt(0).toLowerCase() + str.slice(1);
+
+const kebabToCamelCase = str => str.replace(/-./g, x => x.toUpperCase()[1]);
+
+const uiKitPath = 'src/main/ui-kit';
+const componentName = process.argv[2];
+const componentPath = `${uiKitPath}/${componentName}`;
+const lowerCaseComponentName = firstLetterToLoweCase(componentName);
+const lowerCaseComponentNameCamelCase = firstLetterToLoweCase(kebabToCamelCase(componentName));
+
+const uiKitIndexTsFile = `${uiKitPath}/index.ts`;
+const uiKitIndexScssFile = `${uiKitPath}/index.scss`;
+const componentIndexTsFile = `${componentPath}/index.ts`;
+const componentScssFile = `${componentPath}/styles.scss`;
+
+const componentIndexTsFileContent = `/*
+ * Copyright 2014-2024 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
```

Comment:
Should this text be rendered as comment in a component? If so, maybe put the end date to the variable?

Reply:
Yep, definitely, thanks!